packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx

```diff
@@ -45,7 +45,14 @@ export function _SignInStart(): JSX.Element {
     () => groupIdentifiers(userSettings.enabledFirstFactorIdentifiers),
     [userSettings.enabledFirstFactorIdentifiers],
   );
-  const [identifierAttribute, setIdentifierAttribute] = useState<SignInStartIdentifier>(identifierAttributes[0] || '');
+
+  const onlyPhoneNumberInitialValueExists =
+    !!ctx.initialValues?.phoneNumber && !(ctx.initialValues.emailAddress || ctx.initialValues.username);
+  const shouldStartWithPhoneNumberIdentifier =
+    onlyPhoneNumberInitialValueExists && identifierAttributes.includes('phone_number');
+  const [identifierAttribute, setIdentifierAttribute] = useState<SignInStartIdentifier>(
+    shouldStartWithPhoneNumberIdentifier ? 'phone_number' : identifierAttributes[0] || '',
+  );
```

Comment:
🙃 instead of using `phone_number` hard coded string, could we use `PhoneNumberIdentifier` from `@clerk/types`?